components/select/src/multi-select/selection-list.js

```diff
@@ -44,7 +44,7 @@ const SelectionList = ({ selected, onChange, disabled, options }) => (
                     disabled={isDisabled}
                     marginBottom="0"
                     marginLeft="0"
-                    marginTop="0"
+                    marginTop="4"
```

Comment:
```suggestion
                    marginTop="4px"
```
It probably defaults to px when you don't specify a unit, but I prefer being explicit.